playwright/Customizations/ComplianceWarnings.spec.ts

```diff
@@ -0,0 +1,327 @@
+import { expect } from '@playwright/test';
+import { v4 as uuidv4 } from 'uuid';
+
+import { test } from '../fixtures/customizations';
+import { ensureAuthenticated } from '../helpers/login';
+import { ibFrame, navigateToLandingPage } from '../helpers/navHelpers';
+
+type State =
+  | 'A_only'
+  | 'A_and_B'
+  | 'audit_missing'
+  | 'audit_present'
+  | 'audit_missing_again';
+
+const mkBlueprintListResponse = (
+  blueprintId: string,
+  blueprintName: string,
+) => ({
+  meta: { count: 1 },
+  links: { first: '', last: '' },
+  data: [
+    {
+      id: blueprintId,
+      version: 1,
+      name: blueprintName,
+      description: 'Compliance test blueprint',
+      last_modified_at: new Date().toISOString(),
+    },
+  ],
+});
+
+const mkBlueprintDetails = (
+  blueprintId: string,
+  blueprintName: string,
+  state: State,
+) => {
+  const distribution = 'rhel-9';
+  const image_requests: never[] = [];
```

Comment:
We have a type for image requests, can we use that?

```suggestion
  const image_requests: ImageRequest[] = [];
```